content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -374,7 +374,7 @@ enabled in your cluster. Additionally, it is recommended
 to enable the `PodDisruptionConditions` feature gate in order to be able to detect and handle
 Pod disruption conditions in the Pod failure policy (see also:
 [Pod disruption conditions](/docs/concepts/workloads/pods/disruptions#pod-disruption-conditions)). Both feature gates are
-available in Kubernetes {{< skew currentVersion >}}.
+available in Kubernetes {{< skew currentPatchVersion >}}.
```

Comment:
The original is fine; this statement applies to all patch versions in a minor release.